src/styles/utils/_vars.scss

```diff
@@ -1 +1,5 @@
-$c-gray: #eee;
+$page-color-first: #1d1d1d;
```

Comment:
It will be not clear what color represent variables with names like first, second, third etc. You should use primary, secondary, accent, warning, error etc. Than there's sense in variable. Actually, there's a lot of naming conventions
https://css-tricks.com/what-do-you-name-color-variables/